@xen-orchestra/backups/writers/_MixinBackupWriter.js

```diff
@@ -14,6 +14,10 @@ exports.MixinBackupWriter = (BaseClass = Object) =>
       this._lock = undefined
     }
 
+    _cleanVm(options) {
+      return this._adapter.cleanVm(getVmBackupDir(this._backup.vm.uuid), { ...options, onLog: warn, lock: false })
```

Comment:
IMHO we should catch any error (and log it) to avoid preventing the backup from running.